src/main/kotlin/com/demonwav/mcdev/platform/fabric/inspection/UnresolvedReferenceInspection.kt

```diff
@@ -0,0 +1,60 @@
+/*
+ * Minecraft Dev for IntelliJ
+ *
+ * https://minecraftdev.org
+ *
+ * Copyright (c) 2020 minecraft-dev
+ *
+ * MIT License
+ */
+
+package com.demonwav.mcdev.platform.fabric.inspection
+
+import com.demonwav.mcdev.platform.fabric.util.FabricConstants
+import com.demonwav.mcdev.util.reference.InspectionReference
+import com.intellij.codeInspection.InspectionManager
+import com.intellij.codeInspection.LocalInspectionTool
+import com.intellij.codeInspection.ProblemDescriptor
+import com.intellij.codeInspection.ProblemHighlightType
+import com.intellij.codeInspection.ProblemsHolder
+import com.intellij.json.psi.JsonElementVisitor
+import com.intellij.json.psi.JsonStringLiteral
+import com.intellij.psi.PsiElementVisitor
+import com.intellij.psi.PsiFile
+
+class UnresolvedReferenceInspection : LocalInspectionTool() {
+
+    override fun getStaticDescription() = "Reports unresolved references in Fabric mod JSON files."
+
+    override fun buildVisitor(holder: ProblemsHolder, isOnTheFly: Boolean): PsiElementVisitor {
+        if (holder.file.name == FabricConstants.FABRIC_MOD_JSON) {
+            return Visitor(holder)
+        }
+        return PsiElementVisitor.EMPTY_VISITOR
+    }
+
+    override fun processFile(file: PsiFile, manager: InspectionManager): List<ProblemDescriptor> {
+        if (file.name == FabricConstants.FABRIC_MOD_JSON) {
+            return super.processFile(file, manager)
+        }
+        return listOf()
```

Comment:
```suggestion
        return emptyList()
```
?

Reply:
Coped from a mixin inspection, but sure that works too